README.rst

```diff
@@ -1874,823 +67,14 @@ as in the example below:
-:code:`SchemaGraph` using GraphQL queries.
-
-Cypher query parameters
-~~~~~~~~~~~~~~~~~~~~~~~
-
-RedisGraph `doesn't support query
-parameters <https://github.com/RedisGraph/RedisGraph/issues/544#issuecomment-507963576>`__,
-so we perform manual parameter interpolation in the
-:code:`graphql_to_redisgraph_cypher` function. However, for Neo4j, we can
-use Neo4j's client to do parameter interpolation on its own so that we
-don't reinvent the wheel.
-
-The function :code:`insert_arguments_into_query` does so based on the query
-language, which isn't fine-grained enough here-- for Cypher backends, we
-only want to insert parameters if the backend is RedisGraph, but not if
-it's Neo4j.
-
-Instead, the correct approach for Neo4j Cypher is as follows, given a
-Neo4j Python client called :code:`neo4j_client`:
-
-.. code:: python
-
-    common_schema_info = CommonSchemaInfo(schema, type_equivalence_hints)
-    compilation_result = compile_graphql_to_cypher(common_schema_info, graphql_query)
-    with neo4j_client.driver.session() as session:
-        result = session.run(compilation_result.query, parameters)
-
-FAQ
----
-
-**Q: Do you really use GraphQL, or do you just use GraphQL-like
-syntax?**
-
-A: We really use GraphQL. Any query that the compiler will accept is
-entirely valid GraphQL, and we actually use the Python port of the
-GraphQL core library for parsing and type checking. However, since the
-database queries produced by compiling GraphQL are subject to the
-limitations of the database system they run on, our execution model is
-somewhat different compared to the one described in the standard GraphQL
-specification. See the `Execution model <#execution-model>`__ section
-for more details.
-
-**Q: Does this project come with a GraphQL server implementation?**
-
-A: No -- there are many existing frameworks for running a web server. We
-simply built a tool that takes GraphQL query strings (and their
-parameters) and returns a query string you can use with your database.
-The compiler does not execute the query string against the database, nor
-does it deserialize the results. Therefore, it is agnostic to the choice
-of server framework and database client library used.
-
-**Q: Do you plan to support other databases / more GraphQL features in
-the future?**
-
-A: We'd love to, and we could really use your help! Please consider
-contributing to this project by opening issues, opening pull requests,
-or participating in discussions.
-
-**Q: I think I found a bug, what do I do?**
-
-A: Please check if an issue has already been created for the bug, and
-open a new one if not. Make sure to describe the bug in as much detail
-as possible, including any stack traces or error messages you may have
-seen, which database you're using, and what query you compiled.
-
-**Q: I think I found a security vulnerability, what do I do?**
-
-A: Please reach out to us at graphql-compiler-maintainer@kensho.com so
-we can triage the issue and take appropriate action.
```

Comment:
Do we want to keep some variant of this FAQ around?

Reply:
We do its completely copy pasted in ReadTheDocs.

Reply:
Sorry, I meant in the README itself, since that's what people see when they click on the GitHub repo link.

Reply:
I would prefer to keep the REAMDE slim so that users can quickly get an overview of the library's features. So I am keeping all in-depth tutorial/guides in the ReadTheDocs page and I think that the FAQ is even less "urgent" for me than the top tutorial pages since I nearly always at least go over a couple of tutorial pages before checking the FAQ. So I would prefer not to include the FAQ here. 

Also, none of the top open source projects I've seen, like tensorflow, keras, sqlalchemy and ReadTheDocs, include their FAQ In their README. 

Finally, if the first thing that people want to see after going over a quick overview of the library is the FAQ then maybe we should make the overview better instead.

Thoughts @JulianGoetz ?

Reply:
Yeah, I'm not seeing FAQs in the README for others as well. A not-very-random pass at top open source projects (apache cassandra, kubernetes, ansible, django..) shows that they tend to limit the README to getting started which I think makes sense and seems to be a precedent worth adhering to. Also wow, that Django documentation is thorough. A little intimidating though.. https://docs.djangoproject.com/en/3.1/